docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,81 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+
+Use the segmenter to identify well separated objects above a flat plane.
+Configure the `with_geometries` attribute according to whether you've [configured the frame system](/services/frame-system/#configuration) to provide intrinsic parameters:
+
+- `true`: this segmenter will return point clouds within the `GeometryInFrame` object it captures.
+- `false`: this segmenter will return a single point within the `GeometryInFrame` object it captures.
+
+If `true`, start by [configuring your frame system](/services/frame-system/#configuration) to configure the relative spatial orientation of the components of your robot within Viam's [frame system service](/services/frame-system/).
```

Comment:
I think what this needs to do is explain better what the two options are. You're not choosing whether to configure a parameter to true or false, the reader should choose what functionality they need and based on that set the parameter. Now I am not entirely certain I am capturing the functionality correctly but the way I'd structure it is like this:

```suggestion
This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.

You can use the segmenter to identify well seprated objects above a flat plane or to identify a single object (?):

- To identify multiple well separated objects above a flat plane, set `with_geometries: true`.
   Ensure that you have configured your camera's [`intrinsic_parameters`](https://docs.viam.com/components/camera/calibrate/).
   Then, [configure your frame system](/services/frame-system/#configuration) to configure the relative spatial orientation of the components of your robot within Viam's [frame system service](/services/frame-system/).
   After configuring your frame system, your camera populates its own `Properties` with the intrinsic parameters from the frame system (camera? the next line seems to indicate that?). 
  You can then get those parameters from your camera through the [camera API](/components/camera/#getproperties).
   The segmenter now returns point clouds within the `GeometryInFrame` object it captures.

- To identify a single object above a flat plane, set `with_geometries: false`.
   The segmenter now returns a single point within the `GeometryInFrame` object it captures.
```

I think this way it provides one choice point and only one mention of the frame system. Though I am not sure if the second one also needed the frame system - I guess that is also currently confusing: do I need to configure the frame system in both scenarios or just one?

Reply:
You don't need to/shouldn't have configured the frame system if `false`, but if you do and still set to false, that's fine, you just will get a point back anyways! Is that clear enough now? The parameter determines the behavior of the segmenter-- if you set to `true` and didn't configure anything I assume it will error out